e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/fee-record-correction/feature-flag-enabled/record-correction-log-correct-records.spec.js

```diff
@@ -0,0 +1,151 @@
+import {
+  FEE_RECORD_STATUS,
+  FeeRecordEntityMockBuilder,
+  PENDING_RECONCILIATION,
+  UtilisationReportEntityMockBuilder,
+  RECORD_CORRECTION_REASON,
+  getFormattedMonetaryValue,
+  FeeRecordCorrectionEntityMockBuilder,
+} from '@ukef/dtfs2-common';
+import pages from '../../../../pages';
+import USERS from '../../../../../fixtures/users';
+import { today } from '../../../../../../../e2e-fixtures/dateConstants';
+import { NODE_TASKS } from '../../../../../../../e2e-fixtures';
+import { getMatchingTfmFacilitiesForFeeRecords } from '../../../../../support/utils/getMatchingTfmFacilitiesForFeeRecords';
+
+const bankId = '961';
+const reportId = 1;
+const fee = 50000;
+const correctionId1 = 1;
+const correctionId2 = 2;
+
+const firstReport = UtilisationReportEntityMockBuilder.forStatus(PENDING_RECONCILIATION).withId(reportId).withBankId(bankId).build();
+
+const firstFeeRecord = FeeRecordEntityMockBuilder.forReport(firstReport).withId(1).withStatus(FEE_RECORD_STATUS.TO_DO_AMENDED).build();
+const secondFeeRecord = FeeRecordEntityMockBuilder.forReport(firstReport)
+  .withId(2)
+  .withFacilityId('987654321')
```

Comment:
This only has one correction and yet this value does not match the correction's corrected facility id when it should

Reply:
changed to use a const